src/componentsEventPage/DropDownMenu.js

```diff
@@ -0,0 +1,12 @@
+
+
+export funciton DropDown ({title, list.categorys})
+<div>Category</div>
+<Wrapper>
+    <ul>
+        <li selected >etal</li>
+        <li>sand</li>     
+        <li>medieval</li>
+        <li>others</li>
+    </ul>
+</Wrapper>
```

Comment:
thank you for the feedback, I didnt use this DropDown menu at the moment, so yes, I shouldn't  merge this in this pull requests, because it's another user storie.